crates/biome_css_analyze/tests/specs/nursery/noUnknownUnit/invalid.css.snap

```diff
@@ -655,48 +655,3 @@ invalid.css:26:46 lint/nursery/noUnknownUnit ━━━━━━━━━━━
   
 
 ```
-
-```
-invalid.css:27:31 lint/nursery/noUnknownUnit ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
-
-  ! Unexpected unknown unit: F
-  
-    25 │ a { background: /* comment */ image-set('img1x.png' 1x, 'img2x.png' 2x) left 20x / 15% 60% repeat-x; }
-    26 │ a { background-image: image-set('img1x.png' 1pix, 'img2x.png' 2x); }
-  > 27 │ @font-face { color: U+0100-024F; }
```

Comment:
We have to consider how to handle such an invalid css in the future...

Reply:
I believe that it could be a rule.
Like `noUnknownUnicodeRange` or something?